Makefile

```diff
@@ -8,6 +8,18 @@ TAR := $(shell if which gnutar 1>/dev/null 2> /dev/null; then echo gnutar; else
 
 default: wannier post
 
+PREFIX ?= /usr
+
+install:
+	install -d $(DESTDIR)$(PREFIX)/bin/
+	for x in wannier90.x postw90.x w90chk2chk.x w90spn2spn.x ; do \
+		if [ -f "$$x" ]; then install -Dm755 "$$x" "$(DESTDIR)$(PREFIX)/bin/$$x"; fi; \
```

Comment:
I would remove the `-D` option, as this doesn't exist on the BSD version of `install` (e.g. on Mac OS X) but only on the GNU version (Linux). If I understand correctly it's just to create the folders, but you are already doing it in the line above? I would only leave `-m755`.

Reply:
Thanks! I didn't know this BSD difference, now `-D` are removed.